test/functional/wallet.py

```diff
@@ -57,8 +57,12 @@ def run_test (self):
         assert_equal(len(self.nodes[2].listunspent()), 0)
 
         # Send 21 BTC from 0 to 2 using sendtoaddress call.
+        # Locked memory should use at least 32 bytes to sign each transaction
```

Comment:
Good question. @laanwj, are there any guarantees to `getmemoryinfo` that can be tested? I was under the impression that the locked memory is only used for private keys and thus is really only reserved for wallet. But how do we know it won't get freed?